docs/src/main/asciidoc/modules/ROOT/pages/configuration.adoc

```diff
@@ -66,6 +36,20 @@ The driver supports the following URI schemes, which tweak the security configur
 `bolt` URI schemes are not supported.
 ====
 
+Last but not least, you can add `http` as sub-protocol when using the Neo4j-JDBC driver against a Neo4j instance from 2025.06 on upwards with the Http Query-API enabled:
+
+[source, text]
+
+----
+jdbc:neo4j:http//<host>:<port>/<database>?param1=value1&param2=value2
+jdbc:neo4j:https//<host>:<port>/<database>?param1=value1&param2=value2
+----
+
+Usually, Neo4j provides the HTTP port on 7474 or 7473. When you don't specify the port, the Neo4j JDBC Driver defaults to 80 and 443 for http and https respectively.
+
+WARNING: The query API behaves different in a couple of edge cases. Especially, for big results the driver will materialize them completely and not stream individual records as it does over the binary neo4j protocol.
```

Comment:
Its up to you, but you may want to mention the consequences for no streaming in Query API. For example, if an error would normally occur during streaming, it will fail on query execution immediately when connecting to Query API. However, it should be fine for most users providing they expect errors on query execution as well, which they should.

Reply:
Good points. Thank you.